bootstrap/v2/pkg/kfapp/existing_arrikto/existing.go

```diff
@@ -290,6 +302,72 @@ func getEndpoints(kubeclient client.Client) (string, string, error) {
 	return kfEndpoint, oidcEndpoint, nil
 }
 
+func createSelfSignedCerts(kubeclient client.Client, addr string) error {
```

Comment:
Can we write a unittest? Maybe split up the code into a function to define the K8s v1.Secret object and a function which actaully calls it.

Then its easy to write a unittest to verify the secret is defined as expected.